pyserini/encode/_pseudo.py

```diff
@@ -0,0 +1,38 @@
+#
+# Pyserini: Reproducible IR research with sparse and dense representations
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import json
+
+from pyserini.encode import QueryEncoder
+
+
+class PseudoQueryEncoder(QueryEncoder):
```

Comment:
How about something like `CachedDataQueryEncoder`?

Reply:
Actually, why not just `QueryEncoder.load_data_from_jsonl(...)` or something like that?